lib/main.dart

```diff
@@ -17,7 +20,23 @@ class MyApp extends StatelessWidget {
             textTheme: TextTheme(
               headline: TextStyle(fontSize: 36, fontWeight: FontWeight.bold),
               subhead: TextStyle(fontSize: 17.0, fontWeight: FontWeight.bold),
+              display1: TextStyle(fontSize: 17.0, color: Colors.black),
             )),
         home: Login());
   }
 }
+
+String localhost() {
+  if(Platform.isAndroid) {
+//    return 'http://10.0.2.2:3000';
+    return 'http://192.168.155.135:3000';
```

Comment:
Correct me if I'm wrong, but this will only work on your own computer. Let's adjust it so it'll work on any development server. I found this package, it may help: https://pub.dev/packages/get_ip